### PR TITLE
Splitting doc in multiple files

### PR DESCRIPTION
**Pull request by pakb**
Adding badges for build status
Adding contribution documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +121,6 @@ The target bucket will be defined by the target you've specified (`npm run deplo
 - Only `develop` branch can be deployed at the root of the `dev` bucket.
 - Only `master` branch can be deployed at the root of `int` and `prod` buckets.
 
+## Contributing
 
-### Customize VueCLI configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+See [CONTRIBUTING.md](CONTRIBUTING.md)
```
Comment (ltshb) [inline README.md:126]: Why not removing this and add the link to CONTRIBUTING.md directly on top of README in the Table of Content ?

File: README.md [hunk 2]
```diff
@@ -1,56 +1,22 @@
...
-
-#### Print
-We will redefine the user learning journeys and refresh the approach to Print and Printing grids and annotation
-
-#### 3D nextgen
-We will define the user learning journeys to shadow calculation
 
-#### Draw & measure functions nextgen
-We will define the user learning journeys to shadow snapping and routing
+See [ROADMAP.md](ROADMAP.md)
```
Comment (ltshb) [inline README.md:19]: Why not removing this and add the link to ROADMAP.md directly on top of README in the Table of Content ?

**Review by ltshb — APPROVED** [review]
Just a small suggestion :-)

File: README.md (hunk 1 above)
Comment (pakb) [reply README.md]: fair point, I've rearranged the order of paragraphs

**Pull request merged**